hibike/hibike_tester.py

```diff
@@ -1,34 +1,55 @@
-import hibike_process
-import time
+"""
+Create a separate Hibike process, for testing.
+"""
 from multiprocessing import Process, Pipe, Queue
+# pylint: disable=E0401
```

Comment:
I think the standard for disables now is to actually write the name of the error you are ignoring, rather than the code. This allows better readability later.